net/dnsdist/patches/010-time_t-check.patch

```diff
@@ -0,0 +1,11 @@
+--- a/configure
++++ b/configure
+@@ -5045,7 +5045,7 @@ _ACEOF
+ 
+ 
+ if test $ac_size -lt 8; then :
+-  as_fn_error $? "size of time_t is $ac_size, which is not large enough to fix the y2k38 bug" "$LINENO" 5
++  # as_fn_error $? "size of time_t is $ac_size, which is not large enough to fix the y2k38 bug" "$LINENO" 5
+ fi
```

Comment:
Couldn't you just delete the whole if?

Reply:
Sure - I went for the least amount of typing. Would you prefer I remove the whole if?

Reply:
I removed the whole if.